### ISSUE TITLE
The app should be checked for accessibility using the Accessibility Scanner

### ISSUE THREAD

**Issue by MahdiDavoodi** [MahdiDavoodi/Sievere#7]
There are so many problems with how I handled content descriptions and things related to accessiblitiy. The entire project needs a checkup.
